templates/pioneer-study/example.1.parquetmr.txt

```diff
@@ -0,0 +1,21 @@
+message pioneer-study {
+  required binary creationDate (UTF8);
+  required binary id (UTF8);
+  required binary type (UTF8);
+  required double version;
+  required group payload {
+    required group data {
+        required binary exampleString (UTF8);
+    }
+    required binary pioneerId (UTF8);
+    required binary studyName (UTF8);
+    required double studyVersion;
+  }
+  required group metadata {
+    required int64  Timestamp;
+    required binary documentId (UTF8);
+    optional binary geoCity (UTF8);
+    optional binary geoCountry (UTF8);
+    optional binary normalizedChannel (UTF8);
+  }
+}
```

Comment:
I was thinking something more like this (the schema of the study plus a metadata object): 
```
message pioneer-study-example {
  required group metadata {
    required int64  Timestamp;
    required binary documentId (UTF8);
    required binary pioneerId (UTF8);
    required binary studyName (UTF8);
    required int64 studyVersion;
    optional binary geoCity (UTF8);
    optional binary geoCountry (UTF8);
    optional binary normalizedChannel (UTF8);
  }        
  required binary exampleString (UTF8);
}
```

Reply:
If that's doable, that's indeed much better - no more `payload/data/what-I-really-want` nesting :)